client/Components/TopPackage.jsx

```diff
@@ -3,60 +3,150 @@
               </span>
-              <span className='productDescrAvg'>{props.pReviewAvg < 0.1
+              <span className='b_productDescrAvg'>{props.pReviewAvg < 0.1
                   ? ''
                   : `${props.pReviewAvg}`}</span>
-                <span className='productMiniDescr'>{props.pReviewCounter !== 1
-                    ? <a className='reviewPullDown'>{props.pReviewCounter} Reviews</a>
-                    :  <a className='reviewPullDown'>{props.pReviewCounter} Review}</a>}</span>
+                <span onClick={() => props.scrollToReviewDrawer()} className='b_productMiniDescr'>{props.pReviewCounter !== 1
+                    ?  <a  className='b_reviewPullDown'>{props.pReviewCounter} Reviews</a>
+                    :  <a  className='b_reviewPullDown'>{props.pReviewCounter} Review</a>}
+                </span>
             </div>
             {props.pSoldSeparateMessage !== null
-                ? (<div className='warningDiv'><span className='warningLogo logoProps'/><ul className='warningUl'><li className='warningli'>{props.pSoldSeparateMessage}</li></ul></div>)
-                : null}
-            {props.pColorOpt !== null
-                ? (<div className='optionContainer'>The price reflects selected options.
-                    <div className='option'><span className='optionTitle'><span className='colorOptText'>Colors: </span><span className='optionText'>{colorOptArr[0]}</span></span><span className='openOptions' id='button' onClick={props.displayModal}><svg className='optButton'><path d="M19.71,9.29,18.29,7.88,12,14.17,5.7,7.88,4.29,9.3,12,17Z"/></svg></span></div>
-                   </div>)
+                ? (<div className='b_warningDiv'><span className='b_warningLogo b_logoProps'/><ul className='b_warningUl'><li className='b_warningli'>{props.pSoldSeparateMessage}</li></ul></div>)
                 : null}
+
             {props.pBenefit !== null
-                ? (<div className='benefitDescr'><span>{props.pBenefit} <a className='readMore'> Read More </a></span></div>)
+                ? (<div className='b_benefitDescr'><span>{props.pBenefit} <a className='b_readMore'>Read More</a></span></div>)
                 : null}
 
-            <div id="myModal" className="modal">
+            {colorOptArr !== null ||
+            sizeOptArr !== null ||
+            mattressOptArr !== null ||
+            legsOptArr !== null ||
+            slattedBedBaseOptArr !== null
+            ?
+            (<div className='b_optionContainer'>The price reflects selected options.
+                {colorOptArr !== null
+                    ? (<div className='b_option'><span className='b_optionTitle'><span className='b_colorOptText'>Colors: </span><span className='b_optionText'>{colorOptArr[0]}</span></span><span className='b_openOptions' id='button' onClick={() => props.displayModal('b_myModalColor')}><svg className='b_optButton'><path d="M19.71,9.29,18.29,7.88,12,14.17,5.7,7.88,4.29,9.3,12,17Z"/></svg></span></div>)
```

Comment:
Clean up these element lists, it's fairly hard to read them when they're all in-line like this.  Within the parentheses of the ternary it becomes JSX again and you can use newlines for formatting and readability.